src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/SocketsHttpHandler.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Net.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Net.Http
+{
+    public sealed class SocketsHttpHandler : HttpMessageHandler
+    {
+        public bool UseCookies
+        {
+            get => throw new PlatformNotSupportedException("Property UseCookies is not supported.");
```

Comment:
All of SocketsHttpHandler is unsupported, not just the individual properties.  If we want a message, it's probably best to just say that the whole class is unsupported.  But I also think it's fine to just remove all of these messages.

Reply:
Addressed in recent commits

Reply:
> Addressed in recent commits.

Doesn't appear to be?  I still see the messages specific to each member.